receiver/googlecloudmonitoringreceiver/README.md

```diff
@@ -0,0 +1,58 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@dashpole](https://www.github.com/dashpole), [@TylerHelmuth](https://www.github.com/TylerHelmuth), [@abhishek-at-cloudwerx](https://www.github.com/abhishek-at-cloudwerx) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+The primary objective of the Google Cloud Monitoring Receiver is to gather time series metrics data from all Google services and convert this data into a pipeline format that facilitates further use.
+
+This receiver gets GCP (Google Clout Platform) metrics from [GCP Monitoring REST API] via the [Google SDK for GCP Metrics] and then convert those timeseries data into OTel Format [Pipeline Data].
+
+[GCP Monitoring REST API]: https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.timeSeries/list
+[Google SDK for GCP Metrics]: https://pkg.go.dev/cloud.google.com/go/monitoring/apiv3
+[Pipeline Data]: https://pkg.go.dev/go.opentelemetry.io/collector/pdata
+
+## Configuration
+The following configuration options are supported:
+
+```yaml
+receivers:
+  googlecloudmonitoring:
+    collection_interval: 120s
+    region: us-central1
+    project_id: my-project-id
+    service_account_key: "path/to/service_account.json"
+    services:
+      - service_name: "compute"
+        metric_name: "compute.googleapis.com/instance/cpu/usage_time"
+        delay: 60s
+      - service_name: "connectors"
+        metric_name: "connectors.googleapis.com/flex/instance/cpu/usage_time"
+        delay: 60s
+```
+
+- `collection_interval` (Optional): The interval at which metrics are collected. Default is 60s.
+- `region` (Required): The GCP region where the services are located.
+- `project_id` (Required): The GCP project ID.
+- `service_account_key` (Required): The path to the service account key JSON file.
+- `services` (Required): A list of services to monitor.
+
+Each service can have the following configuration:
+
+- `service_name` (Required): The name of the GCP service (e.g., `compute`).
+- `delay` (Optional): The delay before starting the collection of metrics for this service. Default is 0s.
+- `metric_name` (Optional): The specific metric name to collect. If not set, all metrics are collected.
+
+### Filtering
```

Comment:
Did you forget to remove this section?  It looks like you aren't exposing the raw filter to users.

Reply:
No, metric_name should be included in the raw filter, which is why I categorised it under title filtering.